src/clr-angular/data/datagrid/datagrid-column.ts

```diff
@@ -319,25 +339,38 @@ export class ClrDatagridColumn<T = any> extends DatagridFilterRegistrar<T, Datag
   }
 
   public get filterValue() {
-    return this.filter.value;
+    if (this.filter instanceof DatagridStringFilterImpl || this.filter instanceof DatagridNumericFilterImpl) {
+      return this.filter.value;
+    }
   }
 
   @Input('clrFilterValue')
-  public set updateFilterValue(newValue: string) {
+  public set updateFilterValue(newValue: string | [number, number]) {
     if (!this.filter) {
       return;
     }
-    if (!newValue) {
-      newValue = '';
-    }
-    if (newValue !== this.filter.value) {
-      this.filter.value = newValue;
+    if (this.filter instanceof DatagridStringFilterImpl) {
+      if (!newValue || typeof newValue !== 'string') {
+        newValue = '';
+      }
+      if (newValue !== this.filter.value) {
+        this.filter.value = newValue;
+      }
+    } else if (this.filter instanceof DatagridNumericFilterImpl) {
+      if (!newValue || !(newValue instanceof Array)) {
+        newValue = [null, null];
+      }
+      if (newValue.length === 2 && (newValue[0] !== this.filter.value[0] || newValue[1] !== this.filter.value[1])) {
+        this.filter.value = newValue;
+      }
     }
   }
 
-  public set filterValue(newValue: string) {
-    this.updateFilterValue = newValue;
-    this.filterValueChange.emit(this.filter.value);
+  public set filterValue(newValue: string | [number, number]) {
+    if (this.filter instanceof DatagridStringFilterImpl || this.filter instanceof DatagridNumericFilterImpl) {
```

Comment:
Why was this changed?

Reply:
`value` is not part of `ClrDatagridFilterInterface` but is part of the individual string and numeric filters.  Originally I was going to have the state of the numeric filter be called `values` but I switched it to `value` to match.  Should we make this part of the interface?  I assume any future built-in filter will have some sort of internal state that we can refer to with `value`.

Reply:
Ok, makes sense. I think we can leave it as is. We have some tech debt on making filters more generically supportable but that is a separate task!